src/DeVinci_frontend/helpers/modelStore.js

```diff
@@ -0,0 +1,8 @@
+import { writable } from 'svelte/store';
```

Comment:
this functionality is already implemented in localStorage.js
setLocalFlag
getLocalFlag

better to keep it all there in one place and use the existing functions